.github/workflows/eslint.yml

```
@@ -19,5 +19,7 @@ jobs:
     - name: Yarn
       run: yarn install
 
-    - name: ESLint
-      uses: CatChen/eslint-suggestion-action@v1.1.4
+    - uses: CatChen/eslint-suggestion-action@v1.1.23
```

We can use 1.1.27

```suggestion
    - uses: CatChen/eslint-suggestion-action@v1.1.27
```
or 1.1.29

```suggestion
    - uses: CatChen/eslint-suggestion-action@v1.1.29
```